index.html

```diff
@@ -29,16 +29,17 @@ <h1>Hey there. I’m Imran Hossain <br> <label>I’m a software developer</label
             <div class="contact">
                 <ul>
                     <a href="#">
-                        <li><img src="/images/twitter.svg" alt="twitter"></li>
+                        <li><img src="images/twitter.svg" alt="twitter"></li>
                     </a>
                     <a href="#">
-                        <li><img src="/images/github.svg" alt="github"></li>
+                        <li><img src="images/github.svg" alt="github"></li>
                     </a>
                     <a href="#">
-                        <li><img src="/images/Linkedin icon.svg" alt="linkin"></li>
+                        <li><img src="images/Linkedin icon.svg" alt="linkin"></li>
                     </a>
```

Comment:
[Optional]

- [ ] It's better if you place the `a` tag inside the `li` tag and not the opposite for better semantics 👍